docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -0,0 +1,459 @@
+            "bucketName": "zenkobucket",
+            "bucketMatch": true,
+            "credentialsProfile": "zenko"
+        }
+    },
+    (...)
+
+.. code:: json
+
+    (...)
+    "aws-test": {
+        "type": "aws_s3",
+        "legacyAwsBehavior": true,
+        "details": {
+            "awsEndpoint": "s3.amazonaws.com",
+            "bucketName": "zenkobucket",
+            "bucketMatch": true,
+            "credentials": {
+                "accessKey": "WHDBFKILOSDDVF78NPMQ",
+                "secretKey": "87hdfGCvDS+YYzefKLnjjZEYstOIuIjs/2X72eET"
+            }
+        }
+    },
+    (...)
+
+.. WARNING::
+   If you set :code:`bucketMatch` to :code:`true`, we strongly advise for having
+   only one local bucket per AWS S3 location. Indeed, since your objects names
+   won't be prefixed by the local bucket name in the AWS S3 bucket, you could
+   create inconsistencies seamlessly by putting two objects with the same name
+   in two different local buckets, but since they are ultimately saved in the
+   same AWS bucket, the most recent one would overwrite the earlier one, as the
+   namespace will conflict.
+
+config.json
+^^^^^^^^^^^
+
+Edit the :code:`restEndpoint` section of your :code:`config.json` file to add
+an endpoint definition matching your new AWS S3 hosted region. Following our
+previous example, it would look like:
+
+.. code:: json
+
+    (...)
+        "restEndpoints": {
+        "localhost": "us-east-1",
+        "127.0.0.1": "us-east-1",
+        "cloudserver-front": "us-east-1",
+        "s3.docker.test": "us-east-1",
+        "127.0.0.2": "us-east-1",
+        "s3.amazonaws.com": "aws-test"
+    },
+    (...)
+
+~/.aws/credentials
+^^^^^^^^^^^^^^^^^^
+
+.. TIP::
+   If you set the :code:`credentials` object in your
+   :code:`locationConfig.json` file, you may skip this section
+
+Make sure your :code:`~/.aws/credentials` file has a profile matching the one
+defined in your :code:`locationConfig.json`. Following our previous example, it
+would look like:
+
+
+.. code:: shell
+
+    [zenko]
+    aws_access_key_id=WHDBFKILOSDDVF78NPMQ
+    aws_secret_access_key=87hdfGCvDS+YYzefKLnjjZEYstOIuIjs/2X72eET
+
+Start the server with the ability to write to AWS S3
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Inside the repository, once all the files have been edited, you should be able
+to start the server and start testing pushing to AWS S3.
+
+.. code:: shell
+
+   # Start the server locally
+   $> S3DATA=multiple npm start
+
+Run the server as a docker container with the ability to write to AWS S3
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. TIP::
+   If you set the :code:`credentials` object in your
+   :code:`locationConfig.json` file, you don't need to mount your
+   :code:`.aws/credentials` file
+
+Mount all the files that have been edited to override defaults, and do a
+standard Docker run; then you can start testing pushing to AWS S3.
```

Comment:
you can start writing data to AWS S3 through your CloudServer

Reply:
changed